clients/quarkus/prometheus-client/prometheus-api-spec.yaml

```diff
@@ -0,0 +1,148 @@
+# This is a file maintained by the rhsm-subscriptions project and describes
```

Comment:
Rather than duplicate this file, it's preferable to simply override the path in `clients/build.gradle` like:

```groovy
    openApiGenerate {
        inputSpec = "${rootDir}/clients/prometheus-client/prometheus-api-spec.yaml"
    }
```

(Once we no longer need the spring-boot client, we'd promote `:clients:quarkus:prometheus-client` to `:clients:prometheus-client`, keep the file in its current location, and remove this override).

Reply:
Done